src/com/datacash/training/chapter10/Question7.java

```diff
@@ -0,0 +1,59 @@
+
+package com.datacash.training.chapter10;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Question7 {
+
+	private static final String INPUT_FILE_NAME = "src/com/datacash/training/chapter10/input.txt";
+	private static final String OUTPUT_FILE_NAME = "src/com/datacash/training/chapter10/output.txt";
+
+	public static void main( String[] args ) {
+		FileInputStream inputFile = null;
+		FileOutputStream outputFile = null;
+		try {
+			inputFile = new FileInputStream( INPUT_FILE_NAME );
+
+			List<Integer> output = new ArrayList<Integer>();

```

Comment:
I was originally planning to use some Java 8 features, but that didn't work out so well as I found out further down.